### ISSUE TITLE
[Suggestion] Make Max Stamina a Player Attribute

### ISSUE THREAD

**Issue by SaltSouls** [alRex-U/ParCool#213]
So the title does more of the talking for this one, but I feel like having the max stamina be a player/entity attribute would give modpack creators a lot more ways they can incorporate the stamina system this mod adds into the general game loop. One way I can already see using it, is by making custom potion/effects that increase the max stamina, or using the diet mod to allow players to increase their stamina by eating properly.

TLDR: would make players stamina more versatile and usable by other creators.

**Comment by alRex-U** [comment]
Please let me think.

**Comment by TheTexasDev** [comment]
Just tried this mod out on 1.18.2 and came here to request this same thing!! I have a skill tree mod that uses player attributes, and with the mods I have installed the only configurations I really have access to is sprint speed and jump height.

This mod is already so good and if it used attributes would be even better. Changing max stamina as one thing. But maybe attributes like how many times you can dodge before getting a cooldown, how long that cooldown is, the speed at which you slide down a wall, how long you can hold on to a ledge before it forces you off, how many wall jumps you can preform- you get the idea.

**Comment by alRex-U** [comment]
well, ok I will try it in next update if you all insist

**Comment by PraiseTheSeune** [comment]
Minecraft 1.20.1 + Paraglider
Hello, it seems like the implementation of attributes doesn't work.

I tested it by changing the value with an armor piece which add +800% so I had 9000 stamina (which the attribute value of your mod shows correctly with /attritubte) but the wheel display didn't change (the circle doesn't change) and the rate of the bar consumption was the same (bar %).
So if I get it right the attribute works but interaction with paraglider mod isn't there.

I didn't check the recovery rate yet.
